images/hail-gcp-notebook/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM gcr.io/deeplearning-platform-release/base-cpu
```

Comment:
Do you have a tag to pin against, even if it's explicitly `:latest`?

```suggestion
FROM gcr.io/deeplearning-platform-release/base-cpu:latest
```

Reply:
Yes, good point, thanks! I've pinned the version to `m93`, which is the most recent one.